_posts/2020-10-25-RSE-HEP-SC20-workshop.md

```diff
@@ -0,0 +1,78 @@
+people in such roles; this term originated in the UK in 2012, and has become widely recognized
+worldwide over the past several years.
+
+Although many research projects rely heavily on RSE contributions, formal recognition of the
+unique RSE role is often lacking. Sometimes, these positions are pigeonholed into the existing
+hierarchy of research positions, such as postdoctoral associates and research scientists in
+academia, or analogous positions at other kinds of institutions, including at national labs and in
+industry. However, this classification does not adequately describe RSE responsibilities, and
+can cause problems for RSEs who seek career progression. This lack of a defined position and
+career outlook leads many promising practitioners to abandon research altogether. In
+response, many research institutions are creating specific positions and groups for RSEs, and
+several national organizations of RSEs have formed.
+
+In the HPC community in particular, software projects have historically required significant work
+to develop specialized, highly performant software. This need has increased greatly in the past
+few years, as projects have grappled with adapting their codes to new architectures such as
+GPUs or many-core CPUs. Many projects have also had to address questions of correctness
+and reproducibility. With all of these changes, the need has increased for skilled software
+developers (RSEs) to write HPC software that is understandable, well-tested, maintainable, and
+extensible. Even so, many of these RSEs still struggle to find positions and career recognition
+in the modern research culture.
+
+Because of all this, the time is right to hold a workshop for RSEs in the HPC community at this
+year’s SC20 conference. This workshop will build on RSE panels and BOF sessions at earlier
+SC conferences and other events, and on RSE conferences held in the UK and elsewhere.
+Last year at SC19, for example, we discussed RSE roles and career paths in a BoF; we learned
+that there was a large group interested in the topic, and that the term “RSE” was much better known than it was a couple of years earlier. This workshop will bring together RSEs and allies
+involved in HPC, from all over the world, to grow the RSE community further by establishing and
+strengthening professional networks of current RSEs and RSE leaders.
+
+The planned agenda includes the following talks and discussions:
+
+* Keynotes
+  * *The Birth of RSE* - Simon Hettrick, Software Sustainability Institute
+  * *de-RSE: Forking an RSE Success Story* - Frank Löffler, Friedrich Schiller
+University
+
+* Lightning talks
+  * *How Research Software Engineers Can Support Scientific Software* - Miranda
+Mundt, Sandia National Laboratory
+  * *SC3UIS Research Software Engineering Ecosystem for Scientific Computing* -
+Emilio Carcamo, Universidad Industrial de Santander
+
+* Panel discussion: Building RSE Teams and Groups
+  * Reed Milewicz, Sandia National Laboratory
```

Comment:
My one comment that isn't specific to the text but to the content (since we've shone a light on this recently) is that I hope the speakers and panels are a diverse set of individuals. I'm only seeing two women out of 12, and I can't easily see other kinds of diversity, but I hope that the rest aren't just white, European men.

Reply:
I agree this is an issue - I think it's 3:9 female to male.

Reply:
It should be 4:10 - looks like a couple of names were dropped off between my original and the BSSW version (Bernadette Boscoe and Ben van Werkhoven).  If someone can add those names in, that would be great, or if not I can do it tomorrow.  The full agenda is at http://us-rse.org/rse-hpc-2020/agenda/.

I'll also notify David Bernholdt so that the BSSW version can be fixed.